SIGSEGV on Ubuntu 20

**Describe the bug**

Currently, we are trying to run kepler in a Kubernetes v1.25.9 cluster and ran into a segmentation violation trying to start the exporter.

**To Reproduce**
Steps to reproduce the behavior:
1. Deploy Kepler via [helm chart](https://github.com/sustainable-computing-io/kepler-helm-chart/tree/af1bcd461ef0ed1d008e88a719138b37311885fa) to cluster
2. Wait for exporter to start
3. About one second after "Started Kepler in ...s" the segfault occurs.

**Expected behavior**
Kepler running or providing some useful information about what went wrong and failing gracefully.

**Server**
 - OS: Ubuntu 20 5.4.0-146-generic # 163-Ubuntu SMP Fri Mar 17 18:26:02 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux


**Additional context**
Add any other context about the problem here.
```
I0418 12:10:46.309045       1 gpu.go:43] Failed to init nvml, err: could not init nvml: error opening libnvidia-ml.so.1: libnvidia-ml.so.1: cannot open shared object file: No such file or directory
I0418 12:10:46.312663       1 exporter.go:149] Kepler running on version: d1bfdc4
I0418 12:10:46.312718       1 config.go:169] using gCgroup ID in the BPF program: true
I0418 12:10:46.313033       1 config.go:170] kernel version: 5.4
I0418 12:10:46.313353       1 rapl_msr_util.go:129] failed to open path /dev/cpu/0/msr: no such file or directory
I0418 12:10:46.313684       1 power.go:64] Not able to obtain power, use estimate method
I0418 12:10:46.737341       1 exporter.go:167] Initializing the GPU collector
perf_event_open: No such file or directory
I0418 12:10:48.953114       1 bcc_attacher.go:98] failed to attach perf event cpu_cycles_hc_reader: failed to open bpf perf event: no such file or directory
perf_event_open: Invalid argument
I0418 12:10:48.953203       1 bcc_attacher.go:98] failed to attach perf event cpu_ref_cycles_hc_reader: failed to open bpf perf event: invalid argument
perf_event_open: No such file or directory
I0418 12:10:48.953280       1 bcc_attacher.go:98] failed to attach perf event cpu_instr_hc_reader: failed to open bpf perf event: no such file or directory
perf_event_open: No such file or directory
I0418 12:10:48.953355       1 bcc_attacher.go:98] failed to attach perf event cache_miss_hc_reader: failed to open bpf perf event: no such file or directory
I0418 12:10:48.953376       1 bcc_attacher.go:135] Successfully load eBPF module with option: [-DNUM_CPUS=2]
I0418 12:10:48.977821       1 exporter.go:209] Started Kepler in 2.665289244s
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x100 pc=0xc19c4e]

goroutine 39 [running]:
github.com/sustainable-computing-io/kepler/pkg/cgroup.CCgroupV1StatManager.SetCGroupStat({{0xfed1a8?, 0xc000106f00?}}, {0xc000274b80, 0x40}, 0xc000146c48?)
	/opt/app-root/src/github.com/sustainable-computing-io/kepler/pkg/cgroup/cgroup_stats_linux.go:80 +0xae
github.com/sustainable-computing-io/kepler/pkg/collector/metric.(*ContainerMetrics).UpdateCgroupMetrics(0xc000251b80)
	/opt/app-root/src/github.com/sustainable-computing-io/kepler/pkg/collector/metric/container_metric.go:274 +0x52
github.com/sustainable-computing-io/kepler/pkg/collector.(*Collector).updateCgroupMetrics(0xc00016c5a0)
	/opt/app-root/src/github.com/sustainable-computing-io/kepler/pkg/collector/container_cgroup_collector.go:45 +0x32d
github.com/sustainable-computing-io/kepler/pkg/collector.(*Collector).Update(0xc00016c5a0)
	/opt/app-root/src/github.com/sustainable-computing-io/kepler/pkg/collector/metric_collector.go:116 +0x72
github.com/sustainable-computing-io/kepler/pkg/manager.(*CollectorManager).Start.func1()
	/opt/app-root/src/github.com/sustainable-computing-io/kepler/pkg/manager/manager.go:64 +0x7b
created by github.com/sustainable-computing-io/kepler/pkg/manager.(*CollectorManager).Start
	/opt/app-root/src/github.com/sustainable-computing-io/kepler/pkg/manager/manager.go:56 +0x6a
```

--- a/pkg/cgroup/cgroup_stats_linux.go
+++ b/pkg/cgroup/cgroup_stats_linux.go
@@ -71,8 +71,12 @@ func NewCGroupStatManager(pid int) (CCgroupStatHandler, error) {
 	}
 }
 
+func errPassthrough(err error) error {
+	return err
+}
+
 func (c CCgroupV1StatManager) SetCGroupStat(containerID string, cgroupStatMap map[string]*types.UInt64StatCollection) error {
-	stat, err := c.manager.Stat(cgroups.IgnoreNotExist)
+	stat, err := c.manager.Stat(errPassthrough)
 	if err != nil {
 		return err
 	}
@@ -81,8 +85,7 @@ func (c CCgroupV1StatManager) SetCGroupStat(containerID string, cgroupStatMap ma
 	cgroupStatMap[config.CgroupfsKernelMemory].SetAggrStat(containerID, stat.Memory.Kernel.Usage)
 	cgroupStatMap[config.CgroupfsTCPMemory].SetAggrStat(containerID, stat.Memory.KernelTCP.Usage)
 	// cgroup v1 cpu
-	cgroupStatMap[config.CgroupfsCPU].SetAggrStat(containerID, stat.CPU.Usage.Total/1000) // Usec
-
+	cgroupStatMap[config.CgroupfsCPU].SetAggrStat(containerID, stat.CPU.Usage.Total/1000)        // Usec
 	cgroupStatMap[config.CgroupfsSystemCPU].SetAggrStat(containerID, stat.CPU.Usage.Kernel/1000) // Usec
 	cgroupStatMap[config.CgroupfsUserCPU].SetAggrStat(containerID, stat.CPU.Usage.User/1000)     // Usec
 	// cgroup v1 IO

